networking/build.gradle

```diff
@@ -0,0 +1,28 @@
+apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
+
+group = 'com.github.Wolox'
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion '25.0.2'
+
+    defaultConfig {
+        minSdkVersion 16
+        targetSdkVersion 25
```

Comment:
Nope, that's for regular apps. 
Both `versionCode` and `versionName` are used for app's updates (not library updates, that is specified in the version of the libraries used in the gradle config file for each app).